config/samples/placeholder_nnfstorageprofile.yaml

```diff
@@ -28,7 +28,7 @@ data:
         deactivate: --activate n $VG_NAME
         lockStart: --lock-start $VG_NAME
       vgRemove: $VG_NAME
-      lvCreate: -l 100%VG --stripes $DEVICE_NUM --stripesize=32KiB --name $LV_NAME $VG_NAME
+      lvCreate: --activate ys -l 100%VG --stripes $DEVICE_NUM --stripesize=32KiB --name $LV_NAME $VG_NAME
```

Comment:
This is unrelated, but while you're at it, can you change the `-l` to the longform `--extents` so we match other options?

This line and all lvcreate lines